apps/client/src/features/app-settings/AppSettings.tsx

```diff
@@ -0,0 +1,75 @@
+import { useState } from 'react';
+import { IconButton } from '@chakra-ui/react';
+import { IoClose } from '@react-icons/all-files/io5/IoClose';
+import { ErrorBoundary } from '@sentry/react';
+
+import { useKeyDown } from '../../common/hooks/useKeyDown';
+import { cx } from '../../common/utils/styleUtils';
+
+import { settingPanels, SettingsOption, SettingsOptionId } from './settingsStore';
+
+import style from './AppSettings.module.scss';
+
+interface AppSettingsProps {
+  settings?: SettingsOptionId;
+  onClose: () => void;
+}
+/**
+ * TODO: make store to keep settings
+ *       - whether a tab has unsaved changes
+ */
+
+export default function AppSettings(props: AppSettingsProps) {
+  const { settings = settingPanels[0].id, onClose } = props;
+  useKeyDown(onClose, 'Escape');
+
+  const [selectedTab, setSelectedTab] = useState<SettingsOptionId>(settings);
```

Comment:
Why aren't you using the Chakra-UI `Tabs` component?

Reply:
We consider abandoning Chakra where possible.

Also the amount of customisation we want would take some effort to do, at which point using the Chakra component no longer provides development speed